app/models/access.rb

```diff
@@ -1,17 +1,21 @@
 class Access < ApplicationRecord
   ALLOWED_RESOURCES = %w[Organization FacilityGroup Facility].freeze
   ACTION_TO_ROLE = {
-    view: [:manager, :viewer],
+    manage_overdue_list: [:manager, :health_care_worker, :call_center_worker],
+    view_reports: [:manager, :health_care_worker, :health_administrator],
```

Comment:
Considering that this is the predominant view method in the dashboard, and that PII is checked only in a couple of places (overdue list, patient line list downloads), could we name this `view`?

Reply:
We'd be relying on people to know/remember that `view` = `view_reports` and `view` != `view_pii`. 
Since `view_reports` and `view_pii` are orthogonal, I'd prefer to keep them explicit.

Separate clarification: the overdue list page will actually use the `manage_overdue_list` check, because it allows state changes as well + we wouldn't want to give users with the `call_center` role access to any PII outside of the overdue list.